db/migrate/20231208053131_create_projects.rb

```diff
@@ -0,0 +1,12 @@
+class CreateProjects < ActiveRecord::Migration[7.0]
+  def change
+    create_table :projects do |t|
```

Comment:
One of the mentors will be the Project Owner who will have edit rights to the project. It would be good to have it reflected in the schema